docs/nodes/ps-strategy-heat-capacitor.md

```diff
@@ -0,0 +1,45 @@
+# ps-strategy-heat-capacitor 
+
```

Comment:
In the documentation, please describe the following in a similar way as other nodes:

* Input
* Output
* Dynamic config

You should give a better description of how the output can be used.
One should not have to import the examples to understand how it is used.

Reply:
Todo

Reply:
Da var det gjort :-)